lol-apps/Crypto/Lol/Applications/Benchmarks/KHPRFBenches.hs

```diff
@@ -57,36 +59,44 @@ type Complete3 = 'Intern Complete2 Complete2
 type Complete4 = 'Intern Complete3 Complete3
 type Complete5 = 'Intern Complete4 Complete4
 
-main :: IO ()
-main = do
-  x <- khprfBenches
+main :: forall t . (_) => Proxy t -> IO ()
```

Comment:
Amendment: `main` does not belong here at all. The structure that the SymmSHE benches follow is:

1. Expose several functions returning `rnd [Benchmark]`
2. Combine and call these functions in `lol-apps/Crypto/Lol/Applications/Default.hs`. In this case, the wrapper is trivial because there's only one bench group.
3. Call these combining functions from `main` inside `lol-cpp/benchmarks/BenchAppsCPPMain.hs`